modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/app/components/PageEditor.js

```diff
@@ -95,25 +96,15 @@ function Container({children, item}) {
 	);
 }
 
-function Fragment({item}) {
-	const {fragmentEntryLinks} = useContext(StoreContext);
-
-	const fragmentEntryLink =
-		fragmentEntryLinks[item.config.fragmentEntryLinkId];
-
-	return (
-		<UnsafeHTML className="fragment" markup={fragmentEntryLink.content} />
-	);
-}
-
-function Root({children}) {
-	return <>{children}</>;
-}
-
 function Row({children, item}) {
 	const {layoutData} = useContext(StoreContext);
 	const parent = layoutData.items[item.parentId];
 
+	// eslint-disable-next-line
+	const [{canDrop, isDragging}, drop] = useDrop({
+		accept: [LAYOUT_DATA_ITEM_TYPES.column]
+	});
```

Comment:
will we be able to drop the column too?

Reply:
Yes, you can drop a column into a row.

We're going to work on this in another task, but there are rules that define what can be dropped where.

Reply:
Ok, thanks @julien